setup.py

```diff
@@ -117,7 +117,7 @@ def get_git_sha() -> str:
         "slackclient==2.5.0",  # PINNED! slack changes file upload api in the future versions
         "sqlalchemy>=1.4, <2",
         "sqlalchemy-utils>=0.38.3, <0.39",
-        "sqlparse==0.3.0",  # PINNED! see https://github.com/andialbrecht/sqlparse/issues/562
+        "sqlparse>=0.4.3",
```

Comment:
To minimize risk of breakage caused by backwards incompatible changes:
```suggestion
        "sqlparse>=0.4.3, <0.5",
```